cyclestatus/cycle_status.py

```diff
@@ -180,6 +181,7 @@ async def red_delete_data_for_user(self, *, requester: str, user_id: int) -> Non
     async def _status_add(self, status: str, use_help: bool) -> None:
         status = status.replace(_bot_guild_var, str(len(self.bot.guilds)))
         status = status.replace(_bot_member_var, str(len(self.bot.users)))
+        status = status.replace(_bot_prefix_var, str(await self.bot.get_valid_prefixes()[0]))
         if use_help:
             prefix = (await self.bot.get_valid_prefixes())[0]
             prefix = re.sub(rf"<@!?{self.bot.user.id}>", f"@{self.bot.user.name}", prefix)
```

Comment:
This logic right here (prefix grabbing and replacing the bot mention with `@<botname>`) can be moved up and replace the `.replace(_bot_prefix_var, prefix)`